### PR TITLE
update answers, insert new Q&A

### PR DESCRIPTION
**Pull request by katelynsyu**
Removed FAQ answers that no longer pertain to HOTH 9 (ie. mentorship application form). 
Added some new questions and answers:
- date and duration of HOTH 9
- location (online or offline) - might have to be fixed based on how the school proceeds with Covid and if ResLife ever gets back with an in-person location to host the hackathon
- plugged the discord

### CODE REVIEW

File: src/components/HomePage/FAQSection.js [hunk 1]
```diff
@@ -131,29 +132,56 @@ function FAQSection() {
 		},
 		{
 			question: `What if I don’t have a team?`,
```
Comment (khxia) [inline src/components/HomePage/FAQSection.js:134]: Our current FAQ uses a combination of `'` and `’`. This is probably the fault of the people who made the website in the previous year. However we want to be as consistent as possible. Can you change all instances of  `’` to `'`?

You may not need to use the `&apos` in the template strings, just a `'` should be enough.

File: src/components/HomePage/FAQSection.js [hunk 2]
```diff
@@ -131,29 +132,56 @@ function FAQSection() {
...
+				and it will be 12 hours long.`
+		},
+		{
+			question: `Where is HOTH 9 taking place?`,
+			answer:
+				<>
+					Currently, we are planning for HOTH 9 to be hybrid. A capped number
+					of people can participate in-person, and all in-person attendees must provide
+					a negative Covid test 72 hours before the event. Virtually, the event will be
+					taking place on our ACM Hack discord <Link href='https://discord.gg/gwaaFS2fah'>server</Link>,
```
Comment (khxia) [inline src/components/HomePage/FAQSection.js:151]: `Discord server`
And have the link be for both words.

File: src/components/HomePage/FAQSection.js [hunk 3]
```diff
@@ -131,29 +132,56 @@ function FAQSection() {
...
+			question: `I have coding experience and want to make a difference at HOTH 9. 
 				How can I get involved?`,
 			answer:
 				<>
-					We’d love to have you as a mentor to answer participants’ technical
-					questions and help them with their projects. You can sign up
-					to be a mentor <Link href='https://docs.google.com/forms/d/e/1FAIpQLScPz4jqsMPdZwyXK10Le03SwOiAmiGmh0fBpLOkyrwJMdXmBQ/viewform'>here</Link>!
+					We would love to have you as a mentor to answer participants&apos; technical
+					questions and help them with their projects. If you are interested, send us an email
+					at <Link href='mailto:uclaacmhack@gmail.com'>uclaacmhack@gmail.com!</Link>
```
Comment (khxia) [inline src/components/HomePage/FAQSection.js:176]: Did christeinar confirm that we'll be using the Hack email for mentor applications? I would have thought that the discord server would have been more approachable.

File: src/components/HomePage/FAQSection.js [hunk 4]
```diff
@@ -131,29 +132,56 @@ function FAQSection() {
...
+				</>
+		},
+		{
+			question: `Where can I get updates on HOTH 9 and learn more about what ACM Hack is doing?`,
+			answer:
+				<>
+					Come join us on our discord <Link href='https://discord.gg/gwaaFS2fah'>server</Link>.
```
Comment (khxia) [inline src/components/HomePage/FAQSection.js:183]: Same here for the discord link. 
Also make sure that the discord link that you created is a permanent invite link.

**Review by khxia — CHANGES_REQUESTED** [review]

**Review by khxia — APPROVED** [review]
LGTM. Good job!

Feel free to merge this after Einar has had a look at it and approves.

File: src/components/HomePage/FAQSection.js [hunk 5]
```diff
@@ -112,48 +112,76 @@ AccordionQA.propTypes = {
...
 					come together to create a project from start to finish in a limited time frame.
-					Check out some examples <Link href='https://hoth7.devpost.com/project-gallery'>here</Link>!
+					Check out some examples from last year&apos;s
+					HOTH <Link href='https://hoth8.devpost.com/project-gallery'>here</Link>!
 				</>
 		},
 		{
 			question: `Who can join?`,
-			answer: `Applications for HOTH 8 are open to anyone! However, only current 
+			answer: `Applications for HOTH 9 are open to anyone! However, only current
```
Comment (EinarBalan) [inline src/components/HomePage/FAQSection.js:126]:
```suggestion
		answer: `Applications for HOTH 9 are open to anyone! However, we are 
		prioritizing UCLA students for our prize tracks as well as in person hacking.`
```

File: src/components/HomePage/FAQSection.js [hunk 6]
```diff
@@ -112,48 +112,76 @@ AccordionQA.propTypes = {
...
-				and additional event information. Applications close on February 1st at 11:59pm PST.`
+				the event, you'll be emailed a confirmation form along with appropriate links
+				and additional event information. Applications close on February 21st at 11:59pm PST.`
 		},
 		{
 			question: `What if I have never made a project before?`,
-			answer:	`No worries! Throughout HOTH 8, you’ll have access to workshops on all sorts of
+			answer:	`No worries! Throughout HOTH 9, you'll have access to workshops on all sorts of
 				topics to help you start bringing your ideas to life and mentors to support you through
 				any roadblocks.`
```
Comment (EinarBalan) [inline src/components/HomePage/FAQSection.js:167]: nit: roadblocks to road blocks

File: src/components/HomePage/FAQSection.js (hunk 3 above)
Comment (EinarBalan) [reply src/components/HomePage/FAQSection.js:176]: agreed, we should add a link to the discord here

**Review by EinarBalan — CHANGES_REQUESTED** [review]
Left some comments, but other than that everything looks great! After you push your changes, we'll take one last look and it should be good to merge.

**Review by EinarBalan — APPROVED** [review]
Looks great!

**Pull request merged**